src/prog/main.F90

```diff
@@ -1020,32 +1018,39 @@ subroutine xtbMain(env, argParser)
       end if
 
       if (set%pr_json) then
-         select type (calc)
+         select type (calc)         
          type is (TxTBCalculator)
             call open_file(ich, 'xtbout.json', 'w')
             call main_xtb_json(ich, &
                                mol, chk%wfn, calc%basis, res, fres)
             call close_file(ich)
+
+         type is (TTBLiteCalculator)
+            call open_file(ich, 'tblite.json', 'w')
```

Comment:
Maybe it would be better to try to match the format of xtbout.json here

Reply:
We could print both:)